src/server/game/Chat/Channels/Channel.h

```diff
@@ -164,9 +164,9 @@ class Channel
             if (state) flags |= MEMBER_FLAG_MUTED;
             else flags &= ~MEMBER_FLAG_MUTED;
         }
-        bool IsAllowedToSpeak(uint32 speakDelay) // pussywizard
+        bool IsAllowedToSpeak(uint64 speakDelay) // pussywizard
         {
-            if (lastSpeakTime + speakDelay <= sWorld->GetGameTime())
+            if (lastSpeakTime + speakDelay <= uint64(sWorld->GetGameTime()))
```

Comment:
Okay I assume this is hacky as fuck to make the build work for lastSpeakTime xD (maybe not, I don't know? I am no pro at this)

If this is hack af, how the fuck could I fix this to work xD

Reply:
what about `static_cast<uint64>` instead?

Reply:
I don't know what that does fully, but I read a bit on it. but then you mean
```suggestion
        bool IsAllowedToSpeak(static_cast<uint64>(speakDelay)) // pussywizard
        {
                    if (lastSpeakTime + speakDelay <= static_cast<uint64>(sWorld->GetGameTime()))
```

Reply:
yes